### ISSUE TITLE
[Feature Request] rgb_array don't not support

### ISSUE THREAD

**Issue by wfei1** [PKU-MARL/omnisafe#14]
## Describe the bug

I really appreciate this work, because I don't have to waste time on tedious `mujoco200_linux` installation and use old dependencies to adapt to `safety_gym`, but I noticed that you seem to have some environment codes that is not released, please tell me when you will release it!!!

Also I have a problem when I use it on my server, I want to save the visual mp4 via rgb_array, when I use `render_mode = 'rbg_array'` on safety_gymnasium, this doesn't seem to be supported, will this be supported subsequently?

## Screenshots

<img width="635" alt="image" src="https://user-images.githubusercontent.com/118658162/204095440-12af8734-e39c-4806-a640-ecf65373bfc0.png">

## Checklist

- [x] I have checked that there is no similar issue in the repo. (**required**)
- [x] I have read the [documentation](https://omnisafe.readthedocs.io). (**required**)
- [x] I have provided a minimal working example to reproduce the bug. (**required**)

**Comment by zmsn-2077** [comment]
Your request was handled very seriously and promptly.
We have fixed your problem in Pull Request #15 , you can promptly pull the latest code and try it.

### Screenshots
<img width="1101" alt="image" src="https://user-images.githubusercontent.com/73586554/204100126-00c96f71-e640-44cc-91a8-e52c88757e61.png">

### Test Code
```python
import argparse

import safety_gymnasium


def run_random(env_name):
    env = safety_gymnasium.make(env_name, render_mode='rgb_array')
    obs, _ = env.reset()
    terminated = False
    ep_ret = 0
    ep_cost = 0
    while True:
        print("rgb_array", env.render())
        if terminated:
            print('Episode Return: %.3f \t Episode Cost: %.3f' % (ep_ret, ep_cost))
            ep_ret, ep_cost = 0, 0
            obs, _ = env.reset()
        assert env.observation_space.contains(obs)
        act = env.action_space.sample()
        assert env.action_space.contains(act)
        # Use the environment's built_in max_episode_steps
        if hasattr(env, '_max_episode_steps'):
            max_ep_len = env._max_episode_steps
        obs, reward, cost, terminated, truncated, info = env.step(act)
        ep_ret += reward
        ep_cost += cost


if __name__ == '__main__':

    parser = argparse.ArgumentParser()
    parser.add_argument('--env', default='SafetyPointGoal2-v0')
    args = parser.parse_args()
    run_random(args.env)
```

**Comment by wfei1** [comment]
Thank you so much for your prompt response! I just re-tested it and it's working fine now, thank you very much!
